tests/client/test_v1.py

```diff
@@ -0,0 +1,24 @@
+"""Tests for Shaarli REST API v1 client"""
+from unittest import mock
+
+from shaarli_client.client import ShaarliV1Client
+
+SHAARLI_URL = 'http://domain.tld/shaarli/'
+SHAARLI_SECRET = 's3kr37!'
+
+
+def test_constructor():
+    """Instantiate a new client"""
+    ShaarliV1Client(SHAARLI_URL, SHAARLI_SECRET)
+
+
+@mock.patch('requests.request')
```

Comment:
> calling requests.request() actually does nothing network-wise? Correct?

Correct ;-)

> Is this mock applied to the whole test_v1.py? Or just test_get_info_uri() 

`mock.patch()` decorates the test function, the mock object's scope is the body of the decorated function

The are other possibilities:
- encapsulating tests in a class (`unittests`-style) and mocking all of this class' methods by decorating the class
- using `mock` as a context manager instead of a decorator:
  ```python
  with mock.patch('module.class.some_method') as some_method:
      [do stuff with the mocked callable]
  ```